lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "KeyboardAccessoryView",
```

Comment:
consider adding 'api' to the file name

Reply:
I did not mean to set you as reviewer, 10x :D